core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/model.scala

```diff
@@ -175,7 +180,7 @@ package model {
     rootResponse: RootResponseConfig,
     cors: CORSConfig,
     streams: StreamsConfig,
-    prometheusMetrics: PrometheusMetricsConfig,
+    metrics: Option[MetricsConfig],
```

Comment:
I think this does not need to be an `Option` and I think the code is simpler if it's not an option.

The `reference.conf` ensures that it always has a value at run time with default `enabled=false`, so from user's point of view it is optional.